.github/workflows/test-canary.yml

```diff
@@ -32,7 +32,10 @@ jobs:
           sudo losetup -Dv
           sudo losetup -lv
       - name: "Register QEMU (tonistiigi/binfmt)"
-        run: docker run --privileged --rm tonistiigi/binfmt --install all
+        run: |
```

Comment:
Could you add a comment to explain why "all" can't be used?

Reply:
Of course. Will do.

Reply:
Done